elisp/intero.el

```diff
@@ -1661,6 +1699,9 @@ Automatically performs initial actions in SOURCE-BUFFER, if specified."
       (set-process-query-on-exit-flag process nil)
       (process-send-string process ":set -fobject-code\n")
       (process-send-string process ":set prompt \"\\4\"\n")
+      ;; The following settings give us expanded splices on load
+      (process-send-string process ":set -XTemplateHaskell\n")
+      (process-send-string process ":set -ddump-splices\n")
```

Comment:
(With some GHC API work, we could add support to turn it off explicitly. That can be something we do later.)